Cargo.toml

```diff
@@ -50,7 +50,7 @@ version = "2.0.2"
 
 [dependencies.snarkvm]
 git = "https://github.com/AleoHQ/snarkVM.git"
-rev = "bbc81d0"
+rev = "d6d2f80"
```

Comment:
the huge lockfile diff suggests that maybe another `snarkvm` dependency needs to be updated too?

Reply:
I think so, I'll check this

Reply:
I forgot to change the rev in snarkos-environment. Anyways I'll convert this PR to draft until https://github.com/AleoHQ/snarkVM/pull/1015 is merged.